HW1/P3/P3.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import division
+import numpy as np
+import pyspark
+
+sc = pyspark.SparkContext(appName = "Spark1")
+
+"""
+aa = sc.parallelize([(1,'hi'),(1,'oh')])
+aa = aa.groupByKey().mapValues(list)
+print aa.collect()
+bb = sc.parallelize([(1,'an'),(1,'be')])
+bb = bb.groupByKey().mapValues(list)
+cc = aa.join(bb)
+print cc.collect()
+cc = cc.map(lambda x: (x[0],[item for sublist in x[1] for item in sublist]))
+cc = sc.parallelize(cc.map(lambda x: x[1]).collect()[0])
+print cc.collect()
+print cc.take(2)
+"""
+
+allWords = sc.textFile('A Words.csv').cache()
+allWords = allWords.map(lambda x: (1,x))
+allWords = allWords.groupByKey().mapValues(list)
+bWords = sc.textFile('B Words.csv').cache()
+bWords = bWords.map(lambda x: (1,x))
+bWords = bWords.groupByKey().mapValues(list)
+allWords = allWords.join(bWords)
+allWords = allWords.map(lambda x: (x[0],[item for sublist in x[1] for item in sublist])) # read in data below
+fileNames = ['C Words.csv','D Words.csv','E Words.csv','F Words.csv','G Words.csv','H Words.csv','I Words.csv','J Words.csv','K Words.csv',\
```

Comment:
For example, a simpler solution would have used .union() rather than .join() with a dummy key